src/gui/torrentcontentmodel.cpp

```diff
@@ -50,6 +53,11 @@
 #include "torrentcontentmodelfolder.h"
 #include "torrentcontentmodelfile.h"
 
+#ifdef Q_OS_MAC
+struct NSImage;
+QPixmap qt_mac_toQPixmap(const NSImage *image, const QSizeF &size);
```

Comment:
Also, with the same function for Windows icons, the maintainer decided to utilize the Extras module instead of relaying on the very same exported function. However, for QtMacExtras I see only a function that takes `CGImageRef`. I don't know what are the relations between `NSImage` and `CGImageRef` and is it possible to convert and how costly is the conversion.